README.rst

```diff
@@ -2,11 +2,105 @@ specutils
+-------------------------
+Spectrum1D | Spectrum2D | Spectrum3D
+
+    **Note:**
+    Currently, only 1-dimensional spectra are currently supported
+
+I/O
+-------------------------
+* Reading FITS *simple linear 1D WCS*
+**read_fits_spectrum1d** is the only method required to read various types of FITS formats listed in `this page`_. This method detects the type of WCS, and uses the correct reader accordingly.
+
+.. code-block:: python
+
+    from specutils.io import read_fits
+    myspec = read_fits.read_fits_spectrum1d('myfile.fits')
+
+Currently only linear one-dimensional WCS is implemented, but the examples should give a guide to implement more complex or WCS created from different keywords.
+
+* Reading FITS *multispec format WCS*
+The multispec format holds multiple one-dimensional spectra in a single file. The current 1-D readers can be used to read such a file. A list of spectra is returned whenever the input file is a multispec file.
+
+.. code-block:: python
+
+    from specutils.io import read_fits
+    spectra_list = read_fits.read_fits_spectrum1d('mymultispec.fits')
+    spectra_list[0]
+    Spectrum1D([ 338.06109619,  395.59234619,  326.0012207 , ...,
+                 660.0098877 ,  686.54498291,  689.58374023])
+    spectra_list[1].dispersion
+    <Quantity [ 8293.44875263, 8293.40459999, 8293.36044556,...,
+                8166.53073537, 8166.48250242, 8166.43426803] Angstrom>
+
+* Writing spectra to FITS files
+**write_fits.write** method deciphers the type of object passed automatically and writes spectra to the given file in FITS format. 
+
+.. code-block:: python
+
+    from specutils.io import read_fits, write_fits
+    myspec = read_fits.read_fits_spectrum1d('myfile.fits')
+    write_fits.write(myspec, 'mynewfile.fits')
+
+Writing to a file in multispec FITS format is identical.
+
+World Coordinate System
+-------------------------
+* Spectral 1D WCS
+The simplest WCS for 1D is a lookup table. This is often found in a ascii tables where one column is the lookup table (wavelength array) and one column is the flux array. In terms of the functional transform view: the lookup table represents a parameter (c0):
+
+.. code-block:: python
+
+    from specutils import Spectrum1D
+    from specutils.wcs import specwcs
+    wave = numpy.arange(6000, 9000) # wavelength
+    flux = numpy.random.random(3000) # flux
+    lookup_table_wcs = specwcs.Spectrum1DLookupWCS(wave, unit='angstrom') # Creating the wcs
+    lookup_table_wcs(0) # doing the transformation for pixel 0
+    <Quantity 6000.0 Angstrom>
+    Spectrum1D(flux=flux, wcs=lookup_table_wcs)
+    Spectrum1D([ 0.66118716,  0.39584688,  0.81210479, ...,  0.5238203 ,
+                 0.05986459,  0.11621466])
+
+For more information about the Spectrum1d object go to `Spectrum 1D`_.
+
+Another common WCS is the **linear dispersion** and commonly serialized (encoded) to FITS keyword headers. For linear dispersion we are using the general `Spectrum1DPolynomialWCS`_ WCS.
```

Comment:
It would be helpful to note somewhere which keywords are required for Spectrum1DPolynomialWCS to be successfully read.  Is that in the docs anywhere you can link to?